examples/aws/main.tf

```diff
@@ -32,42 +31,39 @@ resource "gardener_shoot" "test_cluster" {
   }
 
   spec {
-    cloud {
-      profile = "aws"
-      region  = "eu-central-1"
-
-      secret_binding_ref {
-        name = "<secret_binding>"
-      }
+    cloud_profile_name  = "aws"
+    region              = "eu-central-1"
+    secret_binding_name = "<secret_binding>"
 
-      aws {
-        networks {
-          vpc {
-            cidr = "10.250.0.0/16"
-          }
+    networking {
+      type     = "calico"
+      nodes    = "10.250.0.0/19"
+      pods     = "100.96.0.0/11"
+      services = "100.64.0.0/13"
+    }
 
-          internal = ["10.250.112.0/22"]
-          public   = ["10.250.96.0/22"]
-          workers  = ["10.250.0.0/19"]
+    provider {
+      type = "aws"

```

Comment:
spec.provider.infrastructure_config is also must - this part needs to co-work with my PR https://github.com/kyma-incubator/terraform-provider-gardener/pull/59 - i'm tested creation against Canary landscape

```
infrastructure_config {
        aws {
          enableecraccess = true
          networks {
            vpc {
              cidr = "10.50.0.0/16"
            }
            zones {
                name = "us-east-1a"
                internal = "10.50.112.0/22"
                public = "10.50.96.0/22"
                workers = "10.50.0.0/19"
            }
            }
          }
        }
```